libs/domain/cart/totals/src/totals.model.ts

```diff
@@ -0,0 +1,3 @@
+export interface CartTotalsOptions {
+  reference?: string;
```

Comment:
enum with cart/order values?

Reply:
Not sure. The idea was to make it anonymous, and allow to pass any totals related service